### ISSUE TITLE
MetricsTest.java:53-55: Impediment: test for LCOM3 with...

### ISSUE THREAD

**Issue by 0pdd** [yegor256/jpeek#139]
The puzzle `92-8b89e205` from #92 has to be resolved:

https://github.com/yegor256/jpeek/blob/250ed069ee0919634b632e8c454ff150d94e681d/src/test/java/org/jpeek/MetricsTest.java#L53-L55

The puzzle was created by George Aristy on 31-Jan-18. 

Estimate: 30 minutes, role: IMP.

If you have any technical questions, don't ask me, submit new tickets instead. The task will be "done" when the problem is fixed and the text of the puzzle is _removed_ from the source code. Here is more about [PDD](http://www.yegor256.com/2009/03/04/pdd.html) and [about me](http://www.yegor256.com/2017/04/05/pdd-in-action.html).

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by 0crat** [comment]
Job [#139](https://github.com/yegor256/jpeek/issues/139) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
The job [#139](https://github.com/yegor256/jpeek/issues/139) assigned to @llorllale[/z](https://www.0crat.com/u/llorllale), here is [why](https://www.0crat.com/footprint/C7JGJ00DP/462131711). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

**Comment by 0crat** [comment]
@llorllale[/z](https://www.0crat.com/u/llorllale) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by llorllale** [comment]
@0crat waiting on #192

**Comment by 0crat** [comment]
> @0crat waiting on #192  ([here](https://github.com/yegor256/jpeek/issues/139#issuecomment-367445448))

@llorllale The impediment for [#139](https://github.com/yegor256/jpeek/issues/139) was registered successfully by @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0pdd** [comment]
@0pdd all puzzles are solved

**Comment by 0pdd** [comment]
@0pdd all puzzles are solved

**Comment by 0pdd** [comment]
@0pdd all puzzles are solved

**Comment by 0pdd** [comment]
The puzzle `92-8b89e205` has disappeared from the source code, that's why I closed this issue.

**Comment by llorllale** [comment]
@ypshenychka

* This puzzle was closed when PR #198 was merged. 
* Puzzle #228 left for continuing work after #171 is fixed. The above PR involved research that produced more specific instructions on how to solve impediment on #171.

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
`QA` review completed: +8 points just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Comment by 0crat** [comment]
Order was finished, quality was "good": +35 points just awarded to @llorllale[/z](https://www.0crat.com/u/llorllale)

**Comment by 0crat** [comment]
> @0crat quality good ([here](https://github.com/yegor256/jpeek/issues/139#issuecomment-369599384))

@ypshenychka The job [#139](https://github.com/yegor256/jpeek/issues/139) is now out of scope

**Comment by 0pdd** [comment]
@0pdd the only puzzle [#228](https://github.com/yegor256/jpeek/issues/228) is solved here.
